pkg/webhook/resourcedeletionprotection/validating.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023 The Karmada Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcedeletionprotection
+
+import (
+	"context"
+	"fmt"
+	"github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"net/http"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
```

Comment:
```suggestion
	"context"
	"fmt"
	"net/http"

	admissionv1 "k8s.io/api/admission/v1"
	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
	"k8s.io/klog/v2"
	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
	
	"github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
```

Lint is failing, see [this log](https://github.com/karmada-io/karmada/actions/runs/6525003576/job/17717226397?pr=4074#step:5:8)
```
golangci/golangci-lint info installed /home/runner/go/bin/golangci-lint
pkg/webhook/resourcedeletionprotection/validating.go:19: File is not `gci`-ed with --skip-generated -s Standard,Default,Prefix(github.com/karmada-io/karmada) (gci)
	"github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
pkg/webhook/resourcedeletionprotection/validating.go:25: File is not `gci`-ed with --skip-generated -s Standard,Default,Prefix(github.com/karmada-io/karmada) (gci)
	"sigs.k[8](https://github.com/karmada-io/karmada/actions/runs/6525003576/job/17717226397?pr=4074#step:5:9)s.io/controller-runtime/pkg/webhook/admission"
pkg/webhook/resourcedeletionprotection/validating.go:21: File is not `goimports`-ed with -local github.com/karmada-io/karmada (goimports)
```